### PR TITLE
Addition of Pine64 board on supported boards

### PR DESCRIPTION
**Pull request by yeyeto2788**
@makermelissa I'm making this PR in order to have the Pine64 as a supported board on blinka.

Hope it's ok.

Regards.

### CODE REVIEW

File: _blinka/pine64.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- Lithium Battery Port
+- CMOS Sensor Interface Port
+- Optional 802.11GBN + BT 4.0 module expansion module
+- Headphone Jack
+- Micro USB Power (5V 2A) Port
+
+## Purchase
+* [pine64 2GB](https://store.pine64.org/?product=pine-a64-board-2gb)
+* [pine64 1GB](https://store.pine64.org/?product=pine-a64-board-1gb)
+* [pine64 521MB](https://store.pine64.org/?product=pine-a64-board)
```
Comment (makermelissa) [inline _blinka/pine64.md]: This should be 512MB

Comment (yeyeto2788) [reply _blinka/pine64.md]: Hey! 

The link is right https://store.pine64.org/?product=pine-a64-board points directly to the 512MB one.

**Comment by yeyeto2788** [comment]
Sounds good, if you want me to do something else, just let me know.

I'll do the changes now.

File: _blinka/pine64.md (hunk 1 above)
Comment (makermelissa) [reply _blinka/pine64.md]: Yes, but the text says 521MB, which is about 9MB more than it should be ;)

Comment (yeyeto2788) [reply _blinka/pine64.md]: hahahahaha my bad.

**Comment by makermelissa** [comment]
Looks good, but I want to wait until we get the Pine64 boards added before merging this.

**Pull request merged**